src/pages/404.tsx

```diff
@@ -35,20 +33,20 @@ export const getStaticProps = (async ({ locale }) => {
 }) satisfies GetStaticProps<BasePageProps>
 
 const NotFoundPage = () => (
-  <Flex flexDir="column" align="center" w="full" mt={16} mb={0} mx="auto">
-    <Box as={MainArticle} py={4} px={8} w="full">
-      <Heading as="h1" size="2xl" my={8}>
+  <div className="mx-auto mb-0 mt-16 flex w-full flex-col items-center">
+    <div className="w-full px-8 py-4">
```

Comment:
Lets keep using the `MainArticle` component. It has an ID set that help us with the search indexing.

```suggestion
    <MainArticle className="w-full px-8 py-4">
```